### PR TITLE
Feat: split exporters importers

### PR DESCRIPTION
**Pull request by sorodrigo**
A tour through the universe, that ends with companies being splitted into exporters and importers.

Tests are broken, will fix later.

### CODE REVIEW

File: frontend/scripts/react-components/nodes-panel/nodes-panel.selectors.js [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+    return conditions.includes(false);
+  }
+);
+
+export const getCanProceed = createSelector(
+  [getDirtyBlocks, getSomeBlocksNeedUpdate],
+  (dirtyBlocks, someBlocksNeedUpate) => {
+    const mandatoryBlocks = dirtyBlocks.countries && dirtyBlocks.commodities;
+
+    if (mandatoryBlocks && someBlocksNeedUpate === false) {
```
Comment (Bluesmile82) [inline frontend/scripts/react-components/nodes-panel/nodes-panel.selectors.js]: Typo here

File: frontend/scripts/react-components/shared/profile-selector/profile-panel/profile-step-panel/profiles-sources-panel.component.jsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ import Accordion from 'react-components/shared/accordion/accordion.component';
 import ResizeListener from 'react-components/shared/resize-listener.component';
 import { BREAKPOINTS } from 'constants';
 
-import 'react-components/dashboard-element/dashboard-panel/sources-panel.scss';
+// import 'react-components/nodes-panel/country-sources-panel/sources-panel.scss';
```
Comment (Bluesmile82) [inline frontend/scripts/react-components/shared/profile-selector/profile-panel/profile-step-panel/profiles-sources-panel.component.jsx]: Why not just removing it?

File: frontend/scripts/react-components/nodes-panel/countries-panel/countries-panel.scss [hunk 3]
```diff
@@ -0,0 +1,10 @@
+@import 'styles/settings';
+
+.c-countries-panel {
+
+  .country-panel-pill-list {
```
[the remark this answers is not in the document]
Comment (sorodrigo) [reply frontend/scripts/react-components/nodes-panel/countries-panel/countries-panel.scss]: classname was incorrect in countries-panel. fixed it. thanks

File: frontend/scripts/react-components/shared/profile-selector/profile-panel/profile-step-panel/profiles-sources-panel.component.jsx (hunk 2 above)
Comment (sorodrigo) [reply frontend/scripts/react-components/shared/profile-selector/profile-panel/profile-step-panel/profiles-sources-panel.component.jsx]: we need some styles will make a separate file
thanks

**Pull request merged**